duration timeout
bool execute_after_pickup
---
bool success
string message
---
